includes/api/class-wc-admin-rest-reports-coupons-stats-controller.php

```diff
@@ -291,7 +290,7 @@ public function get_collection_params() {
 			),
 			'validate_callback' => 'rest_validate_request_arg',
 		);
-		$params['coupons']  = array(
+		$params['code']  = array(
 			'description'       => __( 'Limit result set to items assigned one or more code.', 'wc-admin' ),
 			'type'              => 'array',
 			'sanitize_callback' => 'wp_parse_slug_list',
```

Comment:
Do you think we should use the coupon ID here since we can reference that directly in the table?  Or are slugs a better approach here?

Reply:
Not sure, implemented it based on swagger, we can switch easily if needed.

Reply:
I'm partial towards using the ID here if possible since it will be consistent with the other endpoints and might make debugging a bit easier if needed in the future.  But I'll leave this up to your judgement.